pages/theme.tsx

```diff
@@ -0,0 +1,297 @@
+import { Button } from '@app/atoms/Button'
+import styled from 'styled-components'
+
+const Box = styled.div`
+  padding: 2rem 0;
+`
+
+const Table = styled.table`
+  border-collapse: collapse;
+  td,
+  th {
+    border: 1rem solid white;
+  }
+  tr:first-child td,
+  tr:first-child th {
+    border-top: 0;
+  }
+  tr:last-child td {
+    border-bottom: 0;
+  }
+  tr td:first-child,
+  tr th:first-child {
+    border-left: 0;
+  }
+  tr td:last-child,
+  tr th:last-child {
+    border-right: 0;
+  }
+
+  td,
+  th {
+    text-align: left;
+  }
+
+  tbody > tr > th {
+    vertical-align: top;
+  }
+`
+
+export default function ThemePage() {
```

Comment:
@ivangabriele What did you want to do with this? Put it in a Storybook at some point?